src/plugins/console/server/lib/spec_definitions/json/generated/indices.simulate_template.json

```diff
@@ -0,0 +1,17 @@
+{
+  "indices.simulate_template": {
```

Comment:
Not checking this one, but I am guessing this will have a body specification? Did not see a corresponding override JSON file spec for the body. I'd say we should probably check through all of the newly added spec and ensure the PUT/POST requests have their body spec included.

Reply:
Ok. Done. This one was needed.